dds/InfoRepo/DCPSInfoRepoServ.cpp

```diff
@@ -41,10 +42,10 @@
 InfoRepo::InfoRepo(int argc, ACE_TCHAR *argv[])
 : ior_file_(ACE_TEXT("repo.ior"))
 , listen_address_given_(0)
-#ifdef DDS_HAS_MINIMUM_BIT
-, use_bits_(false)
-#else
+#if OPENDDS_CONFIG_BUILT_IN_TOPICS
 , use_bits_(true)
+#else
+, use_bits_(false)
```

Comment:
```suggestion
, use_bits_(OPENDDS_CONFIG_BUILT_IN_TOPICS)
```

And remove `#endif` below (GitHub won't make it part of the suggested change).